cypress/support/commands.js

```diff
@@ -324,8 +324,8 @@ Cypress.Commands.add('getModal', () => {
 
 Cypress.Commands.add('getEditor', { prevSubject: 'optional' }, (subject) => {
 	return subject
-		? cy.wrap(subject).find('[data-text-el="editor-container"]')
-		: cy.get('[data-text-el="editor-container"]')
+		? cy.wrap(subject).find('[data-text-el="editor-container"]', { timeout: 20000 })
+		: cy.get('[data-text-el="editor-container"]', { timeout: 20000 })
```

Comment:
Looking at the replay it seems to be not the timeout but somehow viewer is not opening but the file is downloaded instead:

<img width="433" alt="Screenshot 2024-08-14 at 08 44 43" src="https://github.com/user-attachments/assets/03f01ae7-f61d-4313-9841-0b9f03bb577a">